src/components/MyAccount/RequestsTab/RequestsTab.tsx

```diff
@@ -42,7 +42,9 @@ const RequestsTab = ({
     hold ? (
       <Box sx={{ display: "flex", gap: "4px" }}>
         <CancelButton removeHold={removeHold} hold={hold} patron={patron} />
-        {hold.canFreeze && <FreezeButton hold={hold} patron={patron} />}
+        {hold.canFreeze && hold.status === "REQUEST PENDING" && (
```

Comment:
Normally, I'd suggest that this `hold.canFreeze && hold.status === "REQUEST PENDING"` should be a variable so it's easier to read. Such as `{canFreeze && (<FreezeButton ....)}`, but it doesn't seem like that can be done here. NBD but just something to look out for in other contexts.

Reply:
Gotcha . this is sort of a stop gap bc we're reducing the potential statii of a hold to these 3 cases (ready for pickup, request confirmed, request pending), but ideally Sierra (not us) would know/update the `canFreeze` so that if a hold is anything other than pending it'd be false. But because we are not directly reflecting how Sierra treats its holds, we duplicate that condition here